scrypto-derive/src/blueprint.rs

```diff
@@ -222,6 +222,38 @@ pub fn replace_macros(expr: &mut Expr, dependency_exprs: &mut Vec<Expr>) -> Resu
     Ok(())
 }
 
+/// Derive a sensible type identifier from a path, so to pass Radix Engine's constraints.
+pub fn derive_sensible_identifier_from_path(path: &Path) -> Result<String> {
```

Comment:
Perhaps show an example of the transformation in the above documentation just for clarity sake.

Reply:
I meant just in the code doc above :)